engines/python/src/main/java/ai/djl/python/engine/PyProcess.java

```diff
@@ -188,6 +190,7 @@ synchronized void startPythonProcess() {
     }
 
     synchronized void stopPythonProcess(boolean error) {
+        modelLoaded = false;
```

Comment:
I feel move this to startPythonProcess is more readable

Reply:
done